### PR TITLE
cargo-*: add page

### PR DESCRIPTION
**Pull request by acuteenvy**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/cargo-verify-project.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# cargo verify-project
+
+> Check correctness of the `Cargo.toml` manifest and print the result as a JSON object.
```
Comment (gutjuri) [inline pages/common/cargo-verify-project.md:3]:
```suggestion
> Check the correctness of the `Cargo.toml` manifest and print the result as a JSON object.
```

File: pages/common/cargo-verify-project.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-verify-project.html>.
+
+- Check correctness of the current project's manifest:
```
Comment (gutjuri) [inline pages/common/cargo-verify-project.md:6]:
```suggestion
- Check the correctness of the current project's manifest:
```

File: pages/common/cargo-verify-project.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`cargo verify-project`
+
+- Check correctness of the specified manifest file:
```
Comment (gutjuri) [inline pages/common/cargo-verify-project.md:10]:
```suggestion
- Check the correctness of the specified manifest file:
```

File: pages/common/cargo-tree.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Only show dependencies up to the specified depth (e.g. when `n` is 1, display only direct dependencies):
+
+`cargo tree --depth {{n}}`
+
+- Do not display the given package (and its dependencies) in the tree:
+
+`cargo tree --prune {{package_spec}}`
+
+- Show all occurences of repeated dependencies:
```
Comment (kant) [inline pages/common/cargo-tree.md:19]: ocurences --> occurrences

**Review by gutjuri — COMMENTED** [review]
Thanks for your contribution. I have three comments.

**Review by kant — APPROVED** [review]
LGTM

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**